### DISCUSSION TITLE
microG

### DISCUSSION THREAD

**Discussion by pk898kerd** [community/community#128152] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Notification: microG companion version 0.3.0.37525-lh is not supported with App Line version 14.8.2, causing the Line app to rarely notify chats and incoming callers. Please update for me.

**Comment by ebndev** [comment]
Hi @pk898kerd, thanks for being a part of the GitHub Community!

It looks like you're looking for support from [microG](https://microg.org/). You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
